Source/DafnyTestGeneration/Inlining/RemoveShortCircuitRewriter.cs

```diff
@@ -141,7 +142,7 @@ public RemoveShortCircuitingRewriter(Func<MemberDecl, bool> shouldProcessPredica
         return CloneForLoopStmt(forLoopStmt);
       case CallStmt callStmt:
         return CloneCallStmt(callStmt);
-      case PredicateStmt or ForallStmt or RevealStmt: // always ghost
+      case PredicateStmt or ForallStmt or RevealStmt: // always ghost?
```

Comment:
Interesting question. :) Not always ghost, it turns out. At the very least, `ExpectStmt` extends `PredicateStmt` and is not ghost.

Reply:
I see... I guess for now it means that test generation won't try to cover subexpressions within a `PredicateStmt` or a `ForallStmt` but it probably makes sense to add support for that in the future.

Reply:
Yeah, and I think that's probably fine for now.